core/controllers/reader_test.py

```diff
@@ -109,104 +109,6 @@ def test_published_explorations_are_visible_to_logged_in_users(self):
         self.assertEqual(response.status_int, 200)
 
 
-class ReaderControllerEndToEndTests(test_utils.GenericTestBase):
```

Comment:
In retrospect and in lieu of our recent discussions on e2e tests, I'm thinking these should actually be frontend karma tests. Thoughts?